portfolio/src/main/webapp/script.js

```diff
@@ -119,3 +119,21 @@ function togglePause() {
     paused = true;
   }
 }
+
+// function that fetches data from the /data URL
```

Comment:
Prefer /* */ for multi-line comments -- although it seems like this comment could be fit in one line